gx-workspace-update.json

```diff
@@ -0,0 +1,93 @@
+{
```

Comment:
I think we will want to remove this before we merge

Reply:
@Stebalien I want you to handwrite "I will not use 'git add .'" 100 times

Reply:
@whyrusleeping right after I add my updated `.gitignore` with `git add .`.